source/help/apps/desktop-changelog.md

```diff
@@ -0,0 +1,333 @@
+# Desktop Application Changelog
+
+## Release v3.4.0
+
+Expected release date: End of September 2016
```

Comment:
Propose September 30, 2016 instead of "End of September". 

For fans for the product they'll want to know precisely when to expect it. If we're not going to make the date, that's okay, we can just communicate that.